packages/ra-test/README.md

```diff
@@ -100,6 +100,37 @@ it('should send the user to another url', () => {
 });
 ```
 
+### Using the 'renderWithRedux' wrapper function
+
+Instead of wrapping the component under test with the `TestContext` by yourself you can get use of all of the above options and test your components almost natively looking like using just `@testing-library/react` thanks to the `renderWithRedux` wrapper function.
+// to the way using
```

Comment:
I don't get this line

Reply:
My point was to explain that using renderWithRedux is actually doing the same stuff as wrapping your component in TestContext but more convinient. Line 106 is pushed by mistake, it should have been deleted.

Reply:
I have added one more commit where it is deleted